cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/PullCommand.java

```diff
@@ -81,28 +83,32 @@ public void execute() {
 
         String resourceName = argList.get(0);
         String orgName;
+        String packageName;
         String moduleName;
         String version;
 
         // Get org-name
-        int orgNameIndex = resourceName.indexOf("/");
-        if (orgNameIndex != -1) {
-            orgName = resourceName.substring(0, orgNameIndex);
-            if (orgName.equals("ballerina")) {
-                throw LauncherUtils.createLauncherException("`Ballerina` is the builtin organization and its modules"
-                                                                    + " are included in the runtime.");
-            }
-        } else {
-            throw LauncherUtils.createLauncherException("no module-name provided");
+        if (!validateModuleName(resourceName)) {
+            CommandUtil.printError(outStream,
+                    "invalid module name. Provide the module name with the org name ",
```

Comment:
```suggestion
                    "invalid module name. provide the module name with the org name ",
```
Can you please check whether we are using Capital letters for warning commands.

Reply:
Generally, the convention is to start with capital letters followed by a full stop for error messages.